libs/ui-lib/lib/common/components/storage/DisksTable.tsx

```diff
@@ -53,6 +60,77 @@ const SkipFormattingDisk = () => (
   </TextContent>
 );
 
+const getDiskLimitation = (
+  diskName: Disk['name'],
+  hostName: Host['requestedHostname'],
+  holder: Disk,
+) => {
+  if (holder?.driveType === 'LVM') {
+    return `LVM logical volumes were found on the installation disk ${
+      diskName as string
+    } selected for host ${hostName as string} and will be deleted during installation.`;
+  } else if (holder?.driveType === 'RAID') {
+    return `The installation disk ${diskName as string} selected for host ${
+      hostName as string
+    }, is part of a software RAID that will be deleted during the installation.`;
+  } else if (holder?.driveType === 'Multipath') {
+    return `The installation disk ${diskName as string} selected for host ${
+      hostName as string
+    } is managed by multipath. We strongly recommend using the multipath device ${
+      holder.name as string
+    } to improve reliability.`;
+  }
```

Comment:
If `holder` is not defined the code falls through the conditions.
I would recommend first checking if the `holders` value is truthy and then using a switch block on the driveType in order to improve readability.